svgdigitizer/svgplot.py

```diff
@@ -131,14 +132,29 @@
 #  You should have received a copy of the GNU General Public License
 #  along with svgdigitizer. If not, see <https://www.gnu.org/licenses/>.
 # ********************************************************************
+
 import logging
+from enum import Enum
 from functools import cache
+from itertools import groupby
 
 import pandas as pd
 
 logger = logging.getLogger("svgplot")
 
 
+class PageOrientation(Enum):
+    r"""Enum representing the page orientation to be able to to autodetect axis orientation."""
```

Comment:
```suggestion
    r"""
    Page orientation of the source document.
    Used to auto-detect axis orientation.
    """
```

I don't really understand why this is needed. Do we allow inputs to be actually rotated here?

Reply:
Some whole page figures in old papers are in landscape orientation.